extensions/wikia/GlobalWatchlist/GlobalWatchlist.php

```diff
@@ -6,36 +6,29 @@
  *
  * @author Adrian 'ADi' Wieczorek <adi(at)wikia.com>
  * @author Piotr 'Moli' Molski <moli(at)wikia.com>
+ * @author James Sutterfield <james@wikia-inc.com>
  *
  */
 
-if ( !defined( 'MEDIAWIKI' ) ) {
-	echo( "This file is an extension to the MediaWiki software and cannot be used standalone.\n" );
-	exit( 1 ) ;
-}
-
-$wgExtensionCredits['specialpage'][] = array(
+$wgExtensionCredits['specialpage'][] = [
 	'name' => 'GlobalWatchlist',
-	'author' => array('[http://www.wikia.com/wiki/User:Adi3ek Adrian \'ADi\' Wieczorek]', '[http://www.wikia.com/wiki/User:Moli.wikia Piotr Molski]'),
+	'author' => ['[http://www.wikia.com/wiki/User:Adi3ek Adrian \'ADi\' Wieczorek]', '[http://www.wikia.com/wiki/User:Moli.wikia Piotr Molski]'],
 	'descriptionmsg' => 'globalwatchlist-desc',
 	'url' => 'https://github.com/Wikia/app/tree/dev/extensions/wikia/GlobalWatchlist'
-);
-
-// configuration
-$wgGlobalWatchlistMaxDigestedArticlesPerWiki = 50;
```

Comment:
This variable was moved into a class constant inside of GlobalWatchlistBot.class.php.